e2eTest.conf

```diff
@@ -0,0 +1,7 @@
+[Global]
```

Comment:
@chethanv28 Can you delete this file?

Reply:
Thanks for catching this again. I will add this to git ignore locally